app/lib/frontend/templates.dart

```diff
@@ -420,6 +418,34 @@ class TemplateService {
     return values;
   }
 
+  Map<String, dynamic> _renderScoreBars(AnalysisExtract extract) {
+    String renderScoreBar(double score, Brush brush) {
+      return _renderTemplate('pkg/score_bar', {
+        'percent': _formatScore(score ?? 0.0),
```

Comment:
A skipped package will have a 0 score rather than a N/A score?

Reply:
The `percent` will be used only to render the width of the progress bar, while the `score` one line below will be used to render the actual text label. Btw. great catch, because I haven't used the `score` to do so.